### PR TITLE
PKCS#11 - Misc fixes

### PR DESCRIPTION
**Pull request by ruchi393**

### CODE REVIEW

File: ta/pkcs11/src/pkcs11_attributes.c [hunk 1]
```diff
@@ -911,8 +911,9 @@ enum pkcs11_rc check_access_attrs_against_token(struct pkcs11_session *session,
 		return PKCS11_CKR_KEY_FUNCTION_NOT_PERMITTED;
 	}
 
-	if (private && pkcs11_session_is_public(session)) {
-		DMSG("Private object access from a public session");
+	if (private && (pkcs11_session_is_public(session) ||
+			pkcs11_session_is_so(session))) {
+		DMSG("Private object access from a public or so session");
```
Comment (etienne-lms) [inline ta/pkcs11/src/pkcs11_attributes.c:927]: s/so/SO/

**Comment by ruchi393** [comment]
The tests are expected to fail. The change in testcase for IBART to pass has also been submitted on optee-test

**Comment by jforissier** [comment]
@ruchi393 yes chicken-and-egg problem, I suppose this can be merged first (when properly reviewed), then I can restart IBART in the optee_test PR to make sure all is good.

**Review by etienne-lms — COMMENTED** [review]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>` with comment address.

**Comment by etienne-lms** [comment]
> @ruchi393 yes chicken-and-egg problem, I suppose this can be merged first (when properly reviewed), then I can restart IBART in the optee_test PR to make sure all is good.

Either way, I think they are both almost ready to land.

**Comment by ruchi393** [comment]
> `Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>` with comment address.

@etienne-lms , is this for all the patches in the PR ?

**Comment by etienne-lms** [comment]
For the whole series.

**Comment by ruchi393** [comment]
Tag applied after addressing comment

**Review by etienne-lms — APPROVED** [review]

**Pull request merged**